features/appmgt/org.wso2.carbon.appmgt.store.feature/src/main/resources/store/extensions/assets/webapp/pages/asset.jag

```diff
@@ -84,6 +84,11 @@ require('/modules/store.js').exec(function (ctx) {
     var subscriptionService = new subscriptionServiceModule.SubscriptionService();
     subscriptionService.init(jagg, session);
 
+    var username = null;
+    if (authenticator.getLoggedInUser()) {
```

Comment:
Maybe it's a good idea to capture the value of "authenticator.getLoggedInUser()" method invocation and then re-use in the line that follows the if-clause.